### ISSUE TITLE
[v2] TopBar is hidden

### ISSUE THREAD

**Issue by baldarian** [wix/react-native-navigation#3971]
### Issue Description

I've got the simplest stack based navigator. I have also two screens. The first one is registered in stack's children to be rendered initially. In this first screen I don't need TopBar so I've set options in this way to hide it.

```
visible: false,
drawBehind: true
```
When I push a new screen (the second one) from this initially rendered screen (the first one) in that pushed screen the TopBar is invisible, despite the fact that I've set the right options in the second screen to be actually have a visible TopBar with a title.

This issue is related to android. ios works perfectly.

---
### Environment
* React Native Navigation version: 0.56
* React Native version: 2.0.2515
* Platform(s) (iOS, Android, or both?): android
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator / Android 6 / Debug

**Comment by guyca** [comment]
As a workaround, setting `animate: false` in the initial screen should resolve this.

```js
visible: false,
drawBehind: true,
animate: false
```

**Comment by baldarian** [comment]
Yes, the issue related to visibility is gone however using `animate: false` the TopBar behaves somehow buggy. 
Firstly, when I'm navigating to the second screen the title is horizontally centered instead it should be closer to the back button. When I'm pressing the back button to navigate back the title instantly moves to the back button (where it should be initially). I think it looks very ugly.
Is there a way to fix this as well?

Thank you for your reply.

**Comment by guyca** [comment]
Does the second screen have a react component in the TopBar?

**Comment by baldarian** [comment]
Did you mean a custom component for TopBar ? If so, no it doesn't have a component. It just shows a title.

**Comment by guyca** [comment]
@baldarian The TopBar visibility issues should be resolved in 2.0.2554. Please open another issue for the title positioning bug you noticed. I think what you're actually observing is the previous title being set when going back.
